components/text_normalization/fondant_component.yaml

```diff
@@ -9,12 +9,18 @@ consumes:
         type: string
 
 args:
+  remove_additional_whitespaces:
+    description: If true remove all additional whitespace, tabs.
+    type: bool
   apply_nfc:
     description: If true apply nfc normalization
     type: bool
+  remove_bad_patterns:
+    description: If true remove bad patterns
```

Comment:
Could use a better description. It's not clear from this description what bad patterns are.